### PR TITLE
Add some Markdown flourishes and some grammar tweaks

### PR DESCRIPTION
**Pull request by brettcannon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,59 +7,63 @@
...
+    app.main()
+```
 
 
 ## Overview
 
 Appeal is a command-line argument processing library for
 Python, like `argparse`, `optparse`, `getopt`, `click`,
-and `docopt`.  But Appeal takes a refreshing new approach.
+`docopt`, and `Typer`.  But Appeal takes a refreshing new
```
Comment (larryhastings) [inline README.md:65]: Typer!  That's the one I couldn't remember.  Thanks!

File: README.md [hunk 2]
```diff
@@ -78,8 +82,8 @@ you're already halfway to understanding Appeal!
 Appeal isn't like other command-line parsing libraries.
 In fact, you really shouldn't think of Appeal as a
 "command-line parsing library" per se. And, although you
-work with Appeal by defining functions, nor should you
-think of these functions as "callbacks".
+work with Appeal by defining functions, shouldn't you
```
Comment (larryhastings) [inline README.md:85]: "you shouldn't", not "shouldn't you"

**Review by larryhastings — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (brettcannon) [reply README.md:85]:
```suggestion
work with Appeal by defining functions, you shouldn't
```

**Comment by larryhastings** [comment]
Thanks, Brett!  I'm so pleased to get your contribution.  Merry Christmas!

**Pull request merged**